src/staticanalysis/bot/static_analysis_bot/clang/format.py

```diff
@@ -122,8 +123,8 @@ def get_eligible_files():
             groups.append(group)
 
             issues += [
-                ClangFormatIssue(filename, group[0], len(group), revision)
-                for group in groups
+                ClangFormatIssue(filename, g[0], len(group), revision)
```

Comment:
Why the change from `group` to `g`?
I guess you forgot to change `len(group)` to `len(g)`?

Reply:
Because pyflakes was complaining about re-defining the `group` variable from above.